pkg/controller/node/rate_limited_queue_test.go

```diff
@@ -152,25 +152,58 @@ func TestTry(t *testing.T) {
 }
 
 func TestTryOrdering(t *testing.T) {
+	defer func() { now = time.Now }()
+	current := time.Unix(0, 0)
```

Comment:
The addedAt check isn't needed any more (the equivalent exists), and we guarantee that t=0 only matches the "first" entry